Automattic/woocommerce-payments
# Replace the deprecated wcs_renewal_order_meta_query with a new hook

#### Changes proposed in this Pull Request

This PR replaces the use of `wcs_renewal_order_meta_query` which was deprecated in 2.5.0 subscriptions-core. The new hook to use is `wc_subscription_renewal_order_data`. 

#### Testing instructions

- Purchase a subscription using WC Payments. 
- On `trunk` renew the subscription. 
    - Notice you get a deprecation notice saying `wcs_renewal_order_meta_query` is deprecated. 
- On this branch rerun the renewal process
    - Notice the deprecated notice is no longer thrown. 
    - Note that the `_new_order_tracking_complete` meta is still removed from renewal orders.

-------------------


**Post merge**
